sdk/src/types/block/output/delegation.rs

```diff
@@ -359,6 +359,7 @@ impl DelegationOutput {
 
     // Transition, just without full ValidationContext.
     pub(crate) fn transition_inner(current_state: &Self, next_state: &Self) -> Result<(), StateTransitionError> {
+        #[allow(clippy::nonminimal_bool)]
```

Comment:
why allow this? just reverse ! and remove () i think

Reply:
It's a rule from the TIP and I'd like it to read exactly as the TIP